src/schema/nmdc.yaml

```diff
@@ -885,6 +948,31 @@ enums:
 
 
 slots:
+  has_calibration:
+    range: CalibrationInformation
+    description: a calibration instance associated with a process
+    notes: >-
```

Comment:
We should remove these notes in this PR.

Reply:
Thanks! I removed it.